sandbox/stylesheets/default.qss

```diff
@@ -1056,6 +1056,19 @@ appleseed--studio--FoldablePanelWidget > QFrame[hasFrame="true"]
     border-top: 0px;
 }
 
+/*
+ * appleseed::studio:CollapsibleSectionWidget.
+ */
+
+appleseed--studio--CollapsibleSectionWidget > QToolButton
+{
+    border: none;
+}
+
+appleseed--studio--CollapsibleSectionWidget > QScrollArea
+{
+    border: none;
+{
```

Comment:
This should be a closing brace. I'm surprised this works at all (did you test your changes?)

Reply:
Oops. That's interesting :) I tested and it seemingly worked as intended.